src/datanode/src/instance.rs

```diff
@@ -63,15 +63,15 @@ pub struct Instance {
     pub(crate) script_executor: ScriptExecutor,
     pub(crate) table_id_provider: Option<TableIdProviderRef>,
     pub(crate) heartbeat_task: Option<HeartbeatTask>,
-    pub(crate) logstore: Arc<LocalFileLogStore>,
+    pub(crate) logstore: Arc<RaftEngineLogstore>,
```

Comment:
Should there should be a trait obj or type parameter?

Reply:
Using trait object here requires to bind type parameter `Entry` and  `Namespace` to `Instance`, so from the beginning logstore field uses concrete type instead of trait object.